### PR TITLE
Feature/pipeline

### PR DESCRIPTION
**Pull request by tlayh**
Added a pipeline to pugtemplate and fixed a couple of linter issues

### CODE REVIEW

File: module.go [hunk 1]
```diff
@@ -39,6 +39,7 @@ type (
 	}
 )
 
+// Open - open func
```
Comment (dylan-provost) [inline module.go]: If you can get throw the functions, could you add some more details to the comments? For example here, I guess open does a bit more than "open", or at least opens something specific!

Comment (tlayh) [reply module.go]: Good point. Should put more love in it....

File: module.go [hunk 2]
```diff
@@ -47,14 +48,15 @@ func (afs assetFileSystem) Open(path string) (http.File, error) {
 		return nil, err
 	}
 
-	s, err := f.Stat()
+	s, _ := f.Stat()
```
Comment (thebod) [inline module.go]: Please change to
```
s, err := f.Stat()
if err ==nil || s.IsDir {
    return nil, errors.New("not allowed")
```

Comment (tlayh) [reply module.go]: changed it

**Comment by tessig** [comment]
all changes requested by @bastianccm have been done. I will merge in his absence.

**Pull request merged**